apps/web/lib/app-providers.tsx

```diff
@@ -70,22 +70,22 @@ const AppProviders = (props: AppPropsWithChildren) => {
 
   const RemainingProviders = (
     <EventCollectionProvider options={{ apiPath: "/api/collect-events" }}>
-      <ContractsProvider>
-        <SessionProvider session={session || undefined}>
-          <CustomI18nextProvider {...props}>
-            <TooltipProvider>
-              {/* color-scheme makes background:transparent not work which is required by embed. We need to ensure next-theme adds color-scheme to `body` instead of `html`(https://github.com/pacocoursey/next-themes/blob/main/src/index.tsx#L74). Once that's done we can enable color-scheme support */}
-              <ThemeProvider
-                enableColorScheme={false}
-                storageKey={storageKey}
-                forcedTheme={forcedTheme}
-                attribute="class">
-                {props.children}
-              </ThemeProvider>
-            </TooltipProvider>
-          </CustomI18nextProvider>
-        </SessionProvider>
-      </ContractsProvider>
+      {/* <ContractsProvider> */}
```

Comment:
since we are removing metamask we can just remove this code vs commenting it out